.gitignore

```diff
@@ -145,3 +145,4 @@ web/.yarn/*
 !web/.yarn/releases
 !web/.yarn/sdks
 !web/.yarn/versions
+!web/.yarn/**/lib/
```

Comment:
According to the yarn docs (https://next.yarnpkg.com/getting-started/qa#which-files-should-be-gitignored), our yarn related gitignore is correct.
This could also have unwanted effects, like adding a file named "lib" from the `.yarn/cache folder`

IMO it's a better fix to remove line 18 ("lib").
Noted as another problem with using huge auto-generated gitignore templates...